vignettes/typhoid-seroincidence.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Typhoid Seroincidence"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Typhoid Seroincidence}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+#Introduction
+
+This vignette provides users with an example analysis using the *serocalculator* package. Users will be able to determine the seroincidence of typhoid fever in the sample population using existing longitudinal antibody dynamics from ______, and a cross-sectional serosurvey from _____. 
+
+##References
+(To be added)
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+#Sample Analysis
+## ##Load serocalculator package
+##Still in development as of 09/01/23
+The first step in conducting this analysis is to load our necessary packages. Follow the installation instructions [add link] if you ahev not already installed *serocalculator*. 
+
+```{r setup}
+
+#library(devtools)
+#install_github("UCD-SERG/serocalculator")
+library(serocalculator)
+library(tidyverse)
+```
+
+##Load data
+###Load longitudinal parameter mcmc data and prepare for model
+The next step is to load the longitudinal data modeled with Monte Carlo Markov Chain ____ (Add more here). 
+
+These data include the following variables...
+
+The code chunk below loads the data included in the *serocalculator* package. We then create two additional variables, *alpha* and *d*. *Alpha* is the ______ and *d* is the ______. Finally, we select only the variables needed for the analysis. 
+```{r longdata}
+c.hlye.IgG <- 
+  fs::path_package(                          
+  "extdata", 
+  "dmcmc_hlyeigg_09.30.rds", 
+  package = "serocalculator") |> 
+ readRDS()%>%
+  mutate(alpha = alpha*365.25, 
+         d = r-1) %>%
+  select(y1, alpha, d) 
+```
+
+##Load simulated data (lambda = .2) and prepare for model
+The simulated data are assuming a force of infection (FOI, *lambda*) of 0.2. We have selected hlye and IgG as our target measures. From the original dataset, we rename our variables to ______. Finally, we once again limit the dataset to only the variables needed for the analysis. 
+``` {r simdata}
+p.hlye.IgG  <- read_csv("simpophlyeigg.2.csv") %>%
```

Comment:
I think the checks are currently failing due to this line; fix a la https://r-pkgs.org/data.html#sec-data-system-file ?